<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>621</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; font-weight:600; color:#5c3566;&quot;&gt;贪&lt;/span&gt;&lt;span style=&quot; font-size:72pt; font-weight:600; color:#4e9a06;&quot;&gt;吃&lt;/span&gt;&lt;span style=&quot; font-size:72pt; font-weight:600; font-style:italic; color:#ef2929;&quot;&gt;蛇&lt;/span&gt;&lt;span style=&quot; font-size:72pt; font-weight:600; font-style:italic; color:#4e9a06;&quot;&gt;大&lt;/span&gt;&lt;span style=&quot; font-size:72pt; font-weight:600; font-style:italic; color:#5c3566;&quot;&gt;作&lt;/span&gt;&lt;span style=&quot; font-size:72pt; font-weight:600; font-style:italic; color:#ef2929;&quot;&gt;战&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="easy">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>简单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="normal">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>普通</string>
    </property>
   </widget>
   <widget class="QPushButton" name="difficult">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>470</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>困难</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hard">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>460</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>炼狱</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>150</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;上(PgUp)、下(PgDn)、&lt;/p&gt;&lt;p&gt;左(&amp;lt;-)、右(-&amp;gt;)&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>481</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/iamges/image/snake.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>67</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Score：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="tips">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>320</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;重要提示：若想暂停，&lt;/p&gt;&lt;p&gt;请按“P”;&lt;/p&gt;&lt;p&gt;若想重新开始，&lt;/p&gt;&lt;p&gt;请按“O”。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_try">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>500</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; color:#f41010;&quot;&gt;try&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>520</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/iamges/image/finger.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="triangle">
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="score_show">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>70</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>选择游戏难度</string>
     </property>
     <addaction name="action_2"/>
     <addaction name="action_3"/>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
    </widget>
    <addaction name="action_P"/>
    <addaction name="menu_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_P">
   <property name="text">
    <string>返回主界面(&amp;P)</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>简单</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>普通</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>困难</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>炼狱</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
